README.md

```diff
@@ -563,6 +563,39 @@ workspace:
       path: ../my-purescript-facebook
 ```
 
+### I just published a package, how to I use it?
+
+Suppose you published your `newly-published-library` with `spago
+publish`. The package-sets are [rebuilt once a
+day](https://github.com/purescript/registry?tab=readme-ov-file#registry-overview):
+
+> 2. Package sets, which are updated daily and stored in the package-sets directory
+
+Do I need to wait one day to use it?
+
+[No!](https://github.com/purescript/spago/issues/1215)
+
+Just add it into `extraPackages` with the newly released version:
+```yaml
+workspace:
+  packageSet:
+    registry: 41.2.0
+  extraPackages:
+    newly-published-library: 0.1.0
+```
+
+You don't need to add this library to `dependencies`, `extraPackages`
+is enough. You can also publish your package using `extraPackages` in
+this form. Spago will only not let you publish packages which use
+unpublished libraries, i.e. things like
+```yaml
+workspace:
+  extraPackages:
+    newly-published-library:
+	  git: http://.....
+	  ref: ....
+```
+
```

Comment:
I wouldn't really say that "you don't need to add the library to the dependencies": if the library is _not_ a direct dependency of the package to publish (but it's brought in transitively by another package), then this statement is correct but misleading, if it's a direct dependency then I can see it being quite confusing as it might drive people to remove the direct dependency thinking that having the package in the `extraPackages` is enough to include it in the build plan.

The point about git packages is good, but I'd format it as a note to make it stand out:
```suggestion
> [!NOTE]\
> This only works when the package you add to `extraPackages` has been published to the registry. Adding a git dependency will produce an error, as publishing to the Registry only admits build plans that only contain packages coming from the Registry.
```